pkgs/by-name/be/beatsabermodmanager/deps.nix

```diff
@@ -96,4 +96,16 @@
   (fetchNuGet { pname = "Tmds.DBus.Protocol"; version = "0.13.0"; sha256 = "1r5lc8x3iq795l7vnyzs3shvpqz92fldkayvkqmrfjxnqmy0w3kg"; })
   (fetchNuGet { pname = "Tmds.DBus.SourceGenerator"; version = "0.0.2"; sha256 = "03q0fja30216npb3hd39iapr9psriz2k35lymaf4921195y40fiq"; })
   (fetchNuGet { pname = "XamlNameReferenceGenerator"; version = "1.6.1"; sha256 = "0348gj9g5rl0pj2frx4vscj6602gfyn9ba3i1rmfcrxh9jwwa09m"; })
+
+  # HACK: I've manually added these because they are needed to build with sdk 7,
```

Comment:
FYI @Atemu 

If I run `fetch-deps` the package fails to build.  Presumably due to upgrading avalonia?

```
source/BeatSaberModManager/Avalonia.NameGenerator/Avalonia.NameGenerator.AvaloniaNameSourceGenerator/BeatSaberModManager.Views.Pages.IntroPage.g.cs(18,21): error CS0111: Type 'IntroPage' already defines a member called 'InitializeComponent' with the same parameter types [source/BeatSaberModManager/BeatSaberModManager.csproj]
```

Reply:
I sadly have too little experience with dotnet to tell. Thanks a bunch for keeping an eye out and fixing stuff though!